nixos/modules/services/web-apps/firefly-iii.nix

```diff
@@ -22,18 +22,16 @@ let
   env-nonfile-values = filterAttrs (n: v: ! hasSuffix "_FILE" n) cfg.settings;
 
   fileenv-func = ''
+  '';
+
```

Comment:
This isn't used anymore.
```suggestion
```

Reply:
Nice catch. I'd absolutely overlooked it. Thanks!